lib/include/jxl/decode.h

```diff
@@ -236,6 +236,33 @@ typedef enum {
   JXL_DEC_JPEG_RECONSTRUCTION = 0x2000,
 } JxlDecoderStatus;
 
+/** Rewinds decoder to the beginning. The same input must be given again from
+ * the beginning of the file and the decoder will emit events from the beginning
+ * again. When rewinding (as opposed to JxlDecoderReset), the decoder keeps
+ * state about the image, which it can use to skip to a requested frame more
+ * efficiently with JxlDecoderSkipFrames. After rewind,
+ * JxlDecoderSubscribeEvents must be used again, and it is feasible to leave out
```

Comment:
do we gain anything by *requiring* that `JxlDecoderSubscribeEvents` be called again after rewind? could we keep the last value if not?

Reply:
It is more consistent with JxlDecoderReset, and you may want to re-subscribe events because you may want different ones (basic info and color profile are no longer needed anymore). The main gain is there's only one way of doing things, rather than two ways (keeping subscription or changing it)

But I don't feel strongly about it. Are you fine with keeping as is?

Reply:
Changed from 'must' to 'can'